torch/csrc/autograd/engine.h

```diff
@@ -64,6 +64,7 @@ struct TORCH_API Engine {
   std::once_flag start_threads_flag_;
   std::vector<std::shared_ptr<ReadyQueue>> ready_queues_;
```

Comment:
Safe for concurrent access because it is immutable after we initially setup the threads, is that right?

Reply:
Technically, I guess this is protected by `start_threads_flag_`, as that's how we make sure we only initialize the ready queues only once.